util/unifyfs-stage/src/unifyfs-stage.c

```diff
@@ -284,16 +298,19 @@ int main(int argc, char** argv)
         ret = create_status_file(ret);
         if (ret) {
             fprintf(stderr, "failed to create the status file (%s)\n",
-                            strerror(errno));
+                    strerror(errno));
         }
     }
 
-    ret = unifyfs_unmount();
-    if (ret) {
-        fprintf(stderr, "unmounting unifyfs failed (ret=%d)\n", ret);
+    if (should_we_mount_unifyfs) {
+        ret = unifyfs_unmount();
+        if (ret) {
+            fprintf(stderr, "unmounting unifyfs failed (ret=%d)\n", ret);
+        }
     }
-out:
+ out:
     MPI_Finalize();
+ preMPIout:
```

Comment:
Indent

Reply:
Ok.  One space before tags is how emacs formats them by default.  Neither checkpatch, make stylecheck nor Travis said anything about the last several checkins.  Astyle apparent has a different convention.  I take it that astyle is the definitive guide on what's correct, not checkpatch?